tests/test_spatial_attack.py

```diff
@@ -0,0 +1,40 @@
+from typing import List, Tuple
+import pytest
+import eagerpy as ep
+import foolbox as fbn
+import foolbox.attacks as fa
+
+
+def get_attack_id(x: Tuple[fbn.Attack, bool, bool]) -> str:
+    return repr(x[0])
+
+
+# attack
+attacks: List[Tuple[fbn.Attack, bool]] = [
```

Comment:
Remove the `bool` value here, as it is `False` for all tested variants of the attack and the attack never uses gradient information.